app/helpers/ontology_helper.rb

```diff
@@ -13,4 +13,53 @@ def download_path(resource)
   def show_evaluate?
     show_oops? #|| show_foo?
   end
+
+  def status_tag(resource)
+    version = resource.is_a?(Ontology) ? resource.current_version : resource
+    uri = repository_ontology_ontology_version_url(
+      version.ontology.repository, version.ontology, version)
+    html_opts = {
+      class: 'ontology-version-state',
+      data: {
+        ontology_version_id: version.id,
+        uri: uri,
+        state: version.state,
+      }
+    }
+    content_tag(:small, html_opts) do
+      status(version)
+    end
+  end
+
+  def status(resource)
```

Comment:
The same as above seems to apply here. The only usage i could find was the one inside the `status_tag` method, which also seems to be unused.